src/client/AbstractClient.ts

```diff
@@ -216,6 +220,32 @@ class AbstractClient<D extends MetadataDefinition> {
     );
   }
 
+  /**
+   * patch an entity by its id
+   *
+   * @param {string|number} id the entity identifier
+   * @param {ReadonlyArray<PatchOp>} body list of patch operations
+   * @param {Record<string, unknown>} pathParameters path parameters, will be pass to the `getPathBase` method
+   */
+  patch(
+    id: string | number,
+    body: ReadonlyArray<PatchOp> = [],
```

Comment:
Reading rfc 6902, I think that you might name the method something like `jsonPatch` because it is really specific and tied to this RFC and not a PATCH at all.

Reply:
I am not really confident in the status of the RFC 6902 (proposed in 2013, rejected in 2016, erratum in 2017, no activity then… 😨 )

Reply:
Indeed! I missed the differences between PATCH RFCs...
So after discussion, turn out backend planned to use merge patch while I thought we were doing jsonPatch. 
At the end, we think it will be simpler to use the existing update method 😅 .